### DISCUSSION TITLE
Frigate can track vehicles, can it send the location to Home Assistant in real time?

### DISCUSSION THREAD

**Discussion by wxuan1990** [blakeblackshear/frigate#9191] [Ideas]
I want to determine whether I am home or away based on the real-time location of the vehicle. Although this can be determined through multiple zones, setting up multiple zones will display too many events on the user interface, which is not conducive to regular viewing. I want a feature that can send the vehicle’s location to Home Assistant in real-time, or set up a large zone, and then draw several small zones inside it. Only the events of the large zone will be displayed on the Frigate user interface. The events of the small zones are sent directly to Home Assistant and are not displayed on the Frigate user interface.

My English is not very good. I have read the documentation and I feel that this feature has not been considered. So, I want to know if it can be implemented, or if there are other existing methods to achieve my idea

**Comment by NickM-27** [comment]
> Although this can be determined through multiple zones, setting up multiple zones will display too many events on the user interface

Not really, you can already configure which zones create events and which ones are only used for MQTT / home assistant sensors 

https://docs.frigate.video/configuration/zones#restricting-events-to-specific-zones

**Reply by wxuan1990** [reply]
Thank you very much. I have read the webpage of that link several times before, but I didn’t understand it. It turns out that the feature I wanted was always there.I’m sorry for taking up your time
